src/lib.rs

```diff
@@ -995,11 +997,11 @@ impl fmt::LowerHex for Uuid {
     }
 }
 
-impl hash::Hash for Uuid {
-    fn hash<S: hash::Hasher>(&self, state: &mut S) {
-        self.bytes.hash(state)
-    }
-}
+//impl hash::Hash for Uuid {
```

Comment:
We can probably just remove this commented out code.

Reply:
oops